### PR TITLE
@23 Changed version to java 17 to pass build checks.

### PR DESCRIPTION
**Pull request by shreyamaheshwari1**
Hello PA,
changed version back to java 17 for passing build checks.
Kindly Review .

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -57,7 +57,10 @@
 			<artifactId>mysql-connector-j</artifactId>
 			<scope>runtime</scope>
 		</dependency>
-
+		<dependency>
+			<groupId>org.apache.tomcat.embed</groupId>
```
Comment (ajaynegi45) [inline pom.xml:61]: Why add this dependency?

`org.apache.tomcat.embed`

File: src/main/resources/application-development.properties [hunk 2]
```diff
@@ -1,23 +1,36 @@
 ## Database Setup First
 
-## Create a Database and add database name where "Add_Your_Database_Name" is below
-## Make this file as it was earlier before commiting code
-
-# Change this connection string to this format: jdbc:mysql://{ip_address}:{port}/{database_name}
-spring.datasource.url=jdbc:mysql://localhost:3306/Add_Your_Database_Name
+# MySQL Database Connection
```
Comment (ajaynegi45) [inline src/main/resources/application-development.properties:3]: Make this file as it was earlier before commiting code

**Review by ajaynegi45 — CHANGES_REQUESTED** [review]
Hi @shreyamaheshwari1,

Thank you for your prompt work on the issue. I appreciate your dedication to the project.

I have reviewed your work, and I need some changes.

- Mentioned in code Comment.

Once again, thank you for your contribution! ❤️

**Comment by shreyamaheshwari1** [comment]
Sure I'll correct it.

**Pull request closed without merging**